File: app/src/main/java/com/fast_prog/dynate/models/Ride.java
package com.fast_prog.dynate.models;

import java.io.Serializable;

/**
 * Created by sarathk on 1/4/17.
 */

public class Ride implements Serializable {

    //public String vehicleModelId;
    //public String vehicleModSpinnerId;
    //public String vehicleModelName;
    //public String vehicleTypeId;
    //public String vehicleTypeSpinnerId;
    //public String vehicleSizeSpinnerId;
    //public String vehicleSizeName;
    //public String vehicleTypeName;

    public String vehicleSizeId;
    public String pickUpLatitude;
    public String pickUpLongitude;
    public String pickUpLocation;
    public String dropOffLatitude;
    public String dropOffLongitude;
    public String dropOffLocation;

    public boolean isFromSelf;

    public String fromName;
    public String fromMobile;

    public boolean isToSelf;

    public String toName;
    public String toMobile;
    public String subject;
    public String shipment;
    public String date;
    public String hijriDate;
    public String time;
    public String fromISO;
    public String fromMobWithoutISO;
    public String toISO;
    public String toMobWithoutISO;

    public boolean isMessage;

    public String distanceStr;
}

File: app/src/main/java/com/fast_prog/dynate/models/Ride.kt
package com.fast_prog.dynate.models

import java.io.Serializable

/**
 * Created by sarathk on 1/4/17.
 */

class Ride : Serializable {

    var vehicleSizeId: String? = null
    //var vehicleSizeName: String = ""
    var pickUpLatitude: String? = null
    var pickUpLongitude: String? = null
    var pickUpLocation: String? = null
    var dropOffLatitude: String? = null
    var dropOffLongitude: String? = null
    var dropOffLocation: String? = null
    //var isFromSelf: Boolean? = null
    var fromName: String = ""
    var fromMobile: String = ""
    //var isToSelf: Boolean? = null
    var toName: String = ""
    var toMobile: String = ""
    var subject: String = ""
    var shipment: String = ""
    var date: String = ""
    var hijriDate: String = ""
    var time: String = ""
    //var isMessage: Boolean? = null
    var distanceStr: String? = null

    companion object { lateinit var instance: Ride }

    init { instance = this }
}
